### PR TITLE
14013-add-details-to-instruction-about-slack-key-generating-and-using

### PR DESCRIPTION
**Pull request by frolui**

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -109,10 +109,13 @@ clean: ## [FINAL] Cleans the worktree for the next nightly run. Does not clean n
...
-3. The geocint pipeline should send messages to the Slack channel. Create a channel, generate a Slack App and [configure it](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md), add it to a channel, get the Bot User OAuth Token e.g. 'xoxb-111-222-xxxxx' and store it in the `SLACK_KEY` variable in the file `~/.profile`. The angle brackets around your_key need to be removed.
-```shell
-export SLACK_KEY=<your_key>
-```
+3. The geocint pipeline should [send messages](https://api.slack.com/messaging/sending) to the Slack channel. To set slack integration you should:
+
+* Create a [channel](https://slack.com/help/articles/201402297-Create-a-channel);
+* Generate a Slack App and [configure it](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md);
+* Add it to a [channel](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace);
+* Get the Bot User OAuth Token e.g. 'xoxb-111-222-xxxxx'. Bot OAuth Token will be stored in the `SLACK_KEY` variable in the file config.inc.sh. The angle brackets around your_key need to be removed.
```
Comment (amurKontur) [inline DOCUMENTATION.md:117]: are in need to be removed

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -109,10 +109,13 @@ clean: ## [FINAL] Cleans the worktree for the next nightly run. Does not clean n
...
 2. Clone 3 repositories (geocint-runner, geocint-openstreetmap, your repo) to ~/
-3. The geocint pipeline should send messages to the Slack channel. Create a channel, generate a Slack App and [configure it](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md), add it to a channel, get the Bot User OAuth Token e.g. 'xoxb-111-222-xxxxx' and store it in the `SLACK_KEY` variable in the file `~/.profile`. The angle brackets around your_key need to be removed.
-```shell
-export SLACK_KEY=<your_key>
-```
+3. The geocint pipeline should [send messages](https://api.slack.com/messaging/sending) to the Slack channel. To set slack integration you should:
+
+* Create a [channel](https://slack.com/help/articles/201402297-Create-a-channel);
+* Generate a Slack App and [configure it](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md);
+* Add it to a [channel](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace);
```
Comment (amurKontur) [inline DOCUMENTATION.md:116]: "your created channel", not "a channel"

File: DOCUMENTATION.md (hunk 1 above)
Comment (amurKontur) [reply DOCUMENTATION.md:117]: add info how to get Bot User OAuth Token

File: DOCUMENTATION.md (hunk 2 above)
Comment (frolui) [reply DOCUMENTATION.md:116]: i'm not sure that we should make changes here

you should:
* create a channel...

File: DOCUMENTATION.md (hunk 1 above)
Comment (frolui) [reply DOCUMENTATION.md]: Added the link. For me looks like we should have "in" here.

Comment (amurKontur) [reply DOCUMENTATION.md]: fix "are in need to be removed" please too

File: DOCUMENTATION.md (hunk 2 above)
Comment (amurKontur) [reply DOCUMENTATION.md]: Add "Add it to your [channel](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace);"

Comment (amurKontur) [reply DOCUMENTATION.md]: because it is to a random channel, it is the one you created at point 1

File: DOCUMENTATION.md (hunk 1 above)
Comment (amurKontur) [reply DOCUMENTATION.md]: Get the Bot User OAuth Token e.g. 'xoxb-111-222-xxxxx'. Bot OAuth Token will be stored in the `SLACK_KEY` variable in the file config.inc.sh. The angle brackets around your_key are in need to be removed.

Comment (frolui) [reply DOCUMENTATION.md]: replaced

File: DOCUMENTATION.md (hunk 2 above)
Comment (frolui) [reply DOCUMENTATION.md]: fixed

**Review by amurKontur — APPROVED** [review]
now seems ok

**Pull request merged**